introduction/architecture.md

```diff
@@ -110,53 +96,52 @@ while maintaining support for multiple indexes.
 For more on the motivation and design of TimescaleDB, please see our
 [technical blog post][chunking].
 
-## TimescaleDB Clustering (PRIVATE BETA) [](timescaledb-clustering)
+## Distributed Hypertables (PRIVATE BETA) [](distributed-hypertables)
+
+>:WARNING: Distributed hypertables are currently in PRIVATE BETA.
+This feature is not meant for production use. For more information,
+please [contact us][contact].
 
->:WARNING: Running TimescaleDB in a multi-node setup is currently in PRIVATE BETA.
-This method of deployment is not meant for production use. For more information,
-please
-[contact us][contact].
+TimescaleDB supports distributing hypertables across multiple nodes
+(i.e., a cluster) by leveraging the same hypertable and chunk
+primitives as described above. This allows TimescaleDB to scale
+inserts and queries beyond the capabilities of a single TimescaleDB
+instance.
 
-TimescaleDB supports multi-node clustering by leveraging the hypertable and chunk primitives described above.
-In the multi-node topology, all nodes are TimescaleDB instances, but may play different roles
-depending on their responsibilities within a distributed database.
+Distributed hypertables look very similar to regular hypertables, with
+the main difference being that chunks are not stored locally. There
+are also some features of regular hypertables that distributed
+hypertables do not support (see section on current limitations).
 
-### Node Types
+### Distributed Databases and Nodes
+
+A distributed hypertable exist in a *distributed database* that
+consists of multiple databases stored across one or more TimescaleDB
+instances. A database that is part of a distributed database can
+assume one of two *node* roles (but not both):
 
-The two roles that nodes can take on are:
 1) Access node
 2) Data node
 
-A access node accepts all requests from the client, distributes the requests and queries
+A client connects to an access node database. It then distributes the requests and queries
 appropriately to data nodes, and aggregates the results received from the data nodes.
 Access nodes store cluster-wide information about the different data nodes as well as how
 chunks are distributed across those data nodes. Access nodes can also store non-distributed
 hypertables, as well as regular PostgreSQL tables.
 
-Data nodes store data associated with distributed hypertables. Data nodes do not store cluster-wide
-information, and otherwise look just as if they were single TimescaleDB instances. Avoid
-directly accessing local hypertables or chunks of a distributed hypertable on a data node
-as the action can fail or make the distributed hypertable inconsistent. There are
-very few reasons to ever access a data node directly, and all accesses should be done at
-the access node level.
+Data nodes store data associated with a distribute database (i.e., the
+distributed hypertables within it). Data nodes do not store
```

Comment:
Suggestion: "Data nodes store the data of the distributed database." You have already mentioned that a distributed database is a collection of nodes above, so I think this is redundant.